pagefetcher.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python2
+
+import urllib
+import os
+from __builtin__ import str
+
+# set of dictionaries to store url's and names
+CISCO = {"name1": "url1", "name2": "url2"}
```

Comment:
What is the reason for having key-value pairs instead of just the URLs? Wouldn't be easier if it was just
`CISCO = ['url1', 'url2']`?